lib/src/native_updater.dart

```diff
@@ -19,41 +25,70 @@ class NativeUpdater {
   /// Checking New Version
   static displayUpdateAlert({
     @required BuildContext context,
-    @required bool requiresUpdate,
+    @required bool forceUpdate,
     String appStoreUrl,
+    String titlePrefix,
+    String description,
+    String updateButtonLabel,
+    String closeButtonLabel,
+    String ignoreButtonLabel,
   }) async {
     /// Get Current installed version of app
     final PackageInfo info = await PackageInfo.fromPlatform();
```

Comment:
This needs to be removed from here. Is there any relevance to have this call? That should be a user decision and not a plugin decision.

Reply:
Those parameters are already optional.

Reply:
Sorry, my question was why should you keep it? Why is it for? As in, if you remove it, what do you lose?

Reply:
They are all the parameters for using the package. If we remove it, it will break the method call.

Reply:
Sorry, I thought it's all about the highlighted in green. Yes, this `final PackageInfo info = await PackageInfo.fromPlatform();` is required.

Its value from `info` instance required for this:
```
_nativeUpdaterInstance._appName = info.appName;
_nativeUpdaterInstance._packageName = info.packageName;
```

Reply:
We need `appName` for use as a title and `packageName` for use to concatenate Play Store URL, otherwise, it will not work.